docs/guides/cloud-setup-invited.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Set up your IBM Cloud account - invited users
+description: Setup instructions for IBM Quantum on IBM Cloud to submit Qiskit SDK and Qiskit Runtime jobs for users with email invitations
+platform: cloud
+---
+
+{/* cspell:ignore urlendcoded, WQVY */}
+
+# Set up your IBM Cloud account - invited users
+
+If you do not have an invitation from an administrator to join an account, follow the steps in [Set up your IBM Cloud account](/docs/guides/cloud-setup) instead.
+
+You can run workloads on IBM&reg; quantum processing units (QPUs) by setting up an account on IBM Cloud&reg;. You will also need to [install and set up Qiskit and Qiskit Runtime](/docs/guides/install-qiskit#local).
+
+
+Your user account is associated with one or more [instances](./instances), identified by _Cloud Resource Names_ (CRNs) that give access to IBM Quantum services. Additionally, a unique API token (also called a key) is assigned to each account, allowing for IBM Quantum access from Qiskit.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
+
+    <Admonition type="note">
+      The [IBM Quantum Platform dashboard](https://quantum.cloud.ibm.com) and the [Instances page](https://quantum.cloud.ibm.com/instances) list the instances that you can access.
+    </Admonition>
+
+<span id="cloud"></span>
+## Set up to use IBM Cloud
+
+### Before you begin
+
+1. Ensure you are working in an active Python environment with the [Qiskit SDK and Qiskit Runtime installed](/docs/guides/install-qiskit#local).
+1. Accept the email invitation.
+1. Log in with an IBMid or Google account.  If you don't have one, you are guided through creating one. After logging in, confirm your information and accept the invitation to join the account.
+1. Navigate back to [IBM Quantum Platform.](https://quantum.cloud.ibm.com)
+1. Make sure that the correct region is selected in the region switcher in the header. The region controls where your jobs are run and where the job data is kept. You can access either region by using the same API key, but you can only see and access the instances that were created in the region that you're logged in to.
+
+<span id="access-credentials"></span>
+### 1. Find your access credentials
+    1. Find your API key. From the [dashboard](https://quantum.cloud.ibm.com/), create your API key, then copy it to a secure location so you can use it for authentication.
+    2. Optional: Find the instance you want to use from the [Instances](https://quantum.cloud.ibm.com/instances) page. Click the icon to copy its CRN, then save it in a secure location so you can use it to identify the instance.
+
+<span id="cloud-save"></span>
+### 2. Initialize the service (trusted environment)
```

Comment:
should this title be something more like "Connect Qiskit with your service instance" ?